docs/user/dashboard/lens.asciidoc

```diff
@@ -243,3 +248,47 @@ Visualizing saved searches in unsupported in *Lens*.
 ===== How do I change the number of suggestions?
 
 Configuring the *Suggestions* that *Lens* automatically populates is unsupported.
+
+[float]
+[[is-it-possible-to-use-different-indexpatterns-in-lens]]
+===== Can I visualize multiple index patterns in a single visualization?
+
+You can create *Bar*, *Line* and *Area* charts from multiple index patterns.
+
+Each *Layer* in a visualization is associated with an index pattern and mutiple *Layers* can be combined together within the same visualization. Each *Layer* also has a chart switcher button in order to select the best type of visualization for the specific dataset.
+You can also change the index pattern for a single *Layer*.
+
+[float]
+[[why-my-field-x-is-missing-from-the-fields-list]]
+===== Why is my field X missing from the fields list?
```

Comment:
There are a few possible reasons for a field to be missing:

* Full text
* Geo (in all released versions)
* Mapping is not supported by index patterns, like `flattened` or `object`

Also, sometimes user say "missing" when it's actually in the "empty" list. Would be nice to summarize all of these if it can be done concisely, but I don't think it's a blocker.

Reply:
Makes sense. I'll merge this and revisit that particular section in a subsequent PR